modules/ossm-release-3-0-TP-1.adoc

```diff
@@ -0,0 +1,69 @@
+////
+Module included in the following assemblies:
+* service-mesh-docs-main/ossm-release-notes-assembly.adoc
+////
+
+:_mod-docs-content-type: REFERENCE
+[id="ossm-release-3-0-TP-1_{context}"]
+= {SMProductName} 3.0 Technology Preview
+
+This release of {SMProductName} makes it available as Technology Preview.
+
+// Simple table to help clarify for users the component versions they can use with OSSM 3.0 TP1. Likely a more complicated table will be needed for GA. OSSM 3.0 separates all the Operators such as OCP, Istio, Envoy, Kiali, OTEL/Tempo, etc. This is only what is known to work with OSSM 3.0 TP1.
+
+[id="ossm-release-3-0-TP1-component-versions_{context}"]
+== Component versions
+
+|===
+|Component |Version
+
+|{ocp-product-title}
+|14+
+
+|Istio
+|1.23.0
+
+|Envoy Proxy
+|1.31
+
+|Kiali Operator
+|1.89
+
+|Kiali Server
+|1.89
+|===
+
+[id="ossm-release-3-0-TP1-unavailable-features_{context}"]
+== Unavailable features in this release
+
+The following features are not supported in this Technology Preview release:
+
+* {SMPlugin}
+* Istio's Ambient mode
+* Istio's Virtual Machine Architecture
+
+[id="ossm-release-3-0-TP1-istio-default-tag"]
+== Istio `default` tag is unavailable
+
+The Istio `default` tag is unavailable in this release. This means that:
+
+. Validation only works in namespaces with the `istio.io/rev` label.
+. The `istio-injection: enabled` label only works if you name your Istio resource `default`.
```

Comment:
nit: Since Istio here refers to a kube resource, IMO it'd be more clear to use backticks.
```suggestion
. The `istio-injection: enabled` label only works if you name your `Istio` resource `default`.
```